nl_server/util.py

```diff
@@ -14,6 +14,29 @@
 
 import os
 
+from shared.lib.gcs import is_gcs_path
+
+_IS_CUSTOM_DC_ENV_VAR = 'IS_CUSTOM_DC'
```

Comment:
Why use constant if not referenced >1 time?  Adds more hops for the reader of code :)

Reply:
I prefer to declare instead of inlining constants but have removed it here.